lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -2830,9 +2832,12 @@ def register_thread(self, overrides=None):
             "title": "Test Title",
             "body": "Test body",
             "created_at": "2015-05-29T00:00:00Z",
-            "updated_at": "2015-05-29T00:00:00Z"
+            "updated_at": "2015-05-29T00:00:00Z",
+
         })
-        cs_data.update(overrides or {})
+        overrides = overrides.copy() if overrides else {}
+        overrides.setdefault("resp_total", 2)
```

Comment:
Any reason why `resp_total` couldn't be included in the dict for `make_minimal_cs_thread` above?